src/css/video-js.scss

```diff
@@ -50,5 +50,5 @@
   width: 100%;
   height: 100%;
   border: none;
-  visibility: hidden;
+  z-index: -1000;
```

Comment:
Fixes safari and firefox iframe `contextmenu` hijacking

Reply:
(firefox was started to hijack `contextmenu` once we removed `visibility: none`)